knowledge-base/repeat-report-group-names-on-each-page.md

```diff
@@ -0,0 +1,67 @@
+```
+= PageExec("textBox2", Fields.Group) + 
+IIF(PageExec("group", Fields.Value) = PageExec("textBox2", Fields.Value), "", "-Continued ") + 
+IIF(PageExec("textBox2", Fields.Group) = PageExec("textBox2", Last(Fields.Group)), "", " / " + PageExec("textBox2", Last(Fields.Group)))
+```
+On the next _Expression_ line, we determine whether this group is a new one or continues from the previous page. In the latter case, we display "Continued". 
+Note that the first scope is the group name, which assures that the returned value is the first value for the Report group.  
+On the third _Expression_ line, we check whether another group starts on the same page. If so, we display also its name in the Page Header.  
+
+Note that we used the __Last__ [Aggregate Function](../expressions-aggregate-functions). Generally, the __PageExec__ [Page Function](../expressions-page-functions) accepts an aggregate function as second argument. We have skipped the __First__ aggregate function above as it is the default one that would be applied to the Field when an aggregate is omitted. 
+
+The demo report demonstrating this approach can be downloaded from our [Reporting Samples GitHub Repo - ReportWithGroups](https://github.com/telerik/reporting-samples/blob/master/ReportWithGroups/ReportWithGroups.trdp).
+
+### Workaround 2. Display the Group Header Information in the Page Header Only for Groups that Continue from the Previous Page
+For this scenario we have multiple nested Report groups in the Report. We are going to use Group Headers to display the Group information only for the first occurrence of the group instance. Therefore, we keep __[GroupHeader > PrintOnEveryPage](../p-telerik-reporting-groupsection-printoneverypage)__ as __False__. If the group continues we will use the Page Header to display its customized group header information. Here is a sample _Expression_ that would display the grouping field/name followed by "-Continued":
+```
+= IIF(PageExec("group", Fields.Value) = PageExec("textBox2", Fields.Value), "", PageExec("textBox2", Fields.Group) + "-Continued ")
+```
+It is very similar to the second line of the _Expression_ we used in the first approach. Note that you need to use it for each of the nested groups, with the corresopnding group scope and name of the TextBox that displays the group detail data.  
+We need to take caare also of the corner case, when a group starts at the bottom of the previous page without displaying an item there. In this case, the group name won't appear on the second page's Page Header as the first detail of the group will be on this page. To fix this, you need to set all groups' properties [GroupKeepTogether](../t-telerik-reporting-groupkeeptogether) to __FirstDetail__ (the default is _None_).  
```

Comment:
I would change it to "We also have to take care", there are 2 occurrences of "we need"